_layouts/index.html

```diff
@@ -18,32 +18,53 @@ <h1 class="logo">Helm</h1>
       <div id="boat"></div>
     </section>
 
+    <section class="panel -blue" data-purpose="news-and-updates">
+      <div class="news-and-updates">
+        <div class="news-and-updates__item container group">
+          <h2>Helm comes in two great flavors!</h2>
+          <article class="console">
+            <p>
+              The teams at Deis and Google are working hard to bring you a new and improved Helm.
+              In the meanwhile, since so many in the Kubernetes community have come to depend on the
+              original Helm tool, the Deis team is committed to maintaining that original tool as
+              <i>Helm Classic</i>-- new name, same great taste! The name change ensures the original
+              and next generation Helm binaries can exist side-by-side and that the two tools can be
+              discussed unambiguously.
+            </p>
+            <p>
+              Read up on Helm Classic below and stay tuned for updates on
```

Comment:
I'd just shorten this to "For more information and updates about the new Helm check out github.com/kubernetes/helm".